pkg/streamingpromql/engine_test.go

```diff
@@ -49,9 +49,7 @@ func TestUnsupportedPromQLFeatures(t *testing.T) {
 
 	// The goal of this is not to list every conceivable expression that is unsupported, but to cover all the
 	// different cases and make sure we produce a reasonable error message when these cases are encountered.
-	unsupportedExpressions := map[string]string{
-		"absent_over_time(nonexistent{}[1h])": "'absent_over_time' function",
-	}
+	unsupportedExpressions := map[string]string{}
```

Comment:
Should we add `holt_winters` here until we add support for that? Or perhaps an experimental function?

Reply:
Updated in d646f61089ed0868dd997ed45790c881f679cfb8. Let me know if something is incorrect there.